slowloris.py

```diff
@@ -209,10 +191,10 @@ def main():
     for _ in range(socket_count):
         try:
             logging.debug("Creating socket nr %s", _)
-            s = init_socket(ip)
+            s = init_socket(ip, args.port)
         except socket.error as e:
-            logging.debug(e)
-            break
+            logging.error(e)
+            sys.exit(1)
```

Comment:
sys.exit is not good in a loop, things can fail and then recover. Shouldn't require killing the entire script.